docs/DeveloperGuide.md

```diff
@@ -260,6 +250,14 @@ From the diagram illustrated above:
 ### FindPool feature
 This feature allows users to find a pool that contains a passenger with a provided keyword in their name.
 
+Design considerations include the `findPool` command being able to be used in conjunction with the `pool` or `unpool`  command. Due to the limitations of the current iteration of GME, each passenger can only belong to one pool. As the number of passengers and pools increases, it may be hard to keep track of the passengers that exist in pools and those that do not, thus, the user might attempt to create a new pool with a passenger and be informed that that passenger already exists in a pool. The `findPool` feature thus allows the user to find the pool that the passenger already exists in to either confirm if they are in the correctly assigned pool, or `unpool` them if they are not. The `findPool` command works similarly to the `find` command, except that it currently only supports the use of the name prefix: "n/"
```

Comment:
In
_the user might attempt to create a new pool with a passenger and be informed that that passenger already exists in a pool._ 

maybe it should be _that the_